engine/src/test/java/org/camunda/bpm/engine/test/api/history/HistoryServiceTest.java

```diff
@@ -449,6 +450,26 @@ public void testNativeHistoricActivityInstanceTest() {
     assertEquals(1, historyService.createNativeHistoricActivityInstanceQuery().sql("SELECT * FROM " + managementService.getTableName(HistoricProcessInstance.class)).listPage(0, 1).size());
   }
 
+  @Deployment(resources = {"org/camunda/bpm/engine/test/api/oneTaskProcess.bpmn20.xml"})
+  public void testNativeHistoricVariableInstanceTest() {
+    Date date = Calendar.getInstance().getTime();
+    Map<String, Object> vars = new HashMap<String, Object>();
+    vars.put("stringVar", "abcdef");
+    vars.put("dateVar", date);
+    runtimeService.startProcessInstanceByKey(ONE_TASK_PROCESS, vars);
+
+    assertEquals(2, historyService.createNativeHistoricVariableInstanceQuery().sql("SELECT count(*) FROM " + managementService.getTableName(HistoricVariableInstance.class)).count());
+    assertEquals(1, historyService.createNativeHistoricVariableInstanceQuery().sql("SELECT * FROM " + managementService.getTableName(HistoricVariableInstance.class)).listPage(0, 1).size());
+
```

Comment:
It is actually in the next two lines, with the difference, that I assign the result of the query to the variable `variables` (because I need them for additional checks):

```
+    List<HistoricVariableInstance> variables = historyService.createNativeHistoricVariableInstanceQuery().sql("SELECT * FROM " + managementService.getTableName(HistoricVariableInstance.class)).list();
+    assertEquals(2, variables.size()); 
```
Should I add the assertion for plain `list()` in the same format like the others?